pkg/test/utils/testing.go

```diff
@@ -11,10 +12,15 @@ import (
 
 // RunTests runs a Go test method without requiring the Go compiler.
 // This does not currently support test caching.
-func RunTests(testName string, testFunc func(*testing.T)) {
+// If testToRun is set to a non-empty string, it is passed as a `-run` argument to the go test harness.
+func RunTests(testName string, testToRun string, testFunc func(*testing.T)) {
 	// Set the verbose test flag to true since we are not using the regular go test CLI.
 	flag.Lookup("test.v").Value.Set("true")
 
+	if testToRun != "" {
+		flag.Lookup("test.run").Value.Set(fmt.Sprintf("//%s", testToRun))
```

Comment:
Uh I don't understand how is whatever set on flag propagated to the actual run? 🤔 is flag some kind of global context for the whole process? Because the values are not passed in through `testing.MainStart`

Reply:
Added a comment: https://github.com/kudobuilder/kudo/pull/490/commits/c36650f79fc400927cbe6be6f5ebb6408ba4666c

In this PR: https://github.com/kudobuilder/kudo/pull/490